app/views/transfers/tasks/supplemental_funding_agreement/edit.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= render(TaskList::TaskHeaderComponent.new(project: @project, task: @task)) %>
+
+<% content_for :pre_content_nav do %>
+  <% render partial: "shared/back_link", locals: {href: project_tasks_path(@project)} %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for @task, url: project_edit_task_path(@project, @task.class.identifier), method: :put do |form| %>
+      <%= form.govuk_error_summary %>
+
+      <div class="govuk-form-group">
+        <h2 class="govuk-heading-l"><%= t("transfer.task.supplemental_funding_agreement.clear_section.title") %></h2>
+        <div class="app-task-section__hint">
+          <%= t("transfer.task.supplemental_funding_agreement.clear_section.hint.html") %>
+        </div>
+
+        <%= govuk_details(
+              summary_text: t("conversion.task.supplemental_funding_agreement.clear_section.guidance_link"),
```

Comment:
Looks like you are using `conversion` instead of `transfer` for your locale string. Same as the one below too

Reply:
Oops cut & paste fail 😆